src/neo4j-arc/graph-visualization/GraphVisualizer/Graph/visualization/Visualization.ts

```diff
@@ -385,15 +385,15 @@ export class Visualization {
     )
   }
 
-  zoomInClick(): void {
-    this.handleZoomClick(ZoomType.IN)
+  handleZoomIn(): void {
```

Comment:
Could we call them simply `zoomIn` rather than `handleZoomIn` (since they're not really handling anything)